slices/golang-1.22-go.yaml

```
@@ -0,0 +1,133 @@
+package: golang-1.22-go
+
+essential:
+  - golang-1.22-go_copyright
+
+slices:
+  bins:
+    essential:
+      - golang-1.22-go_core-libs
+      - golang-1.22-go_go-env
+    contents:
+      /usr/lib/go-1.22/bin/go:
+      /usr/lib/go-1.22/bin/gofmt:
+
+  core-libs:
+    essential:
+      - golang-1.22-go_cmd-src
+      - golang-1.22-go_core-tools
+      - golang-1.22-go_go-build-src
+      - golang-1.22-go_headers
+      - golang-1.22-go_internal-src
+      - golang-1.22-go_runtime-src
+      - golang-1.22-go_symlinks
+      - golang-1.22-go_time-src
+      - golang-1.22-src_src-with-tests
+
+  # Note: The following tools slices are grouped according to their
+  # functionalities listed in https://pkg.go.dev/cmd.
+  core-tools:
+    contents:
+      /usr/lib/go-1.22/pkg/tool/linux_*/asm:
+      /usr/lib/go-1.22/pkg/tool/linux_*/compile:
+      /usr/lib/go-1.22/pkg/tool/linux_*/link:
+      # vet is used for go test
+      /usr/lib/go-1.22/pkg/tool/linux_*/vet:
+
+  cgo-tools:
+    v3-essential:
+      cpp-13-x86-64-linux-gnu_cc1: {arch: [amd64]}
+      cpp-13-aarch64-linux-gnu_cc1: {arch: [arm64]}
+    essential:
+      - binutils_assembler
+      - binutils_linker
+      - gcc_gcc
+      - libbinutils_libs
+      - libc6-dev_core
+      - libc6-dev_libresolv
+      - libc6-dev_runtime
+      - libgcc-13-dev_core
+      - libgcc-13-dev_headers
```

I think this can be simplified to:

```suggestion
      - gcc_gcc
      - libc6-dev_core
      - libc6-dev_libresolv
      - libc6-dev_runtime
      - libgcc-13-dev_core
```

---

Fixed in 4b58540. Even the `libgcc-13-dev_core` is ditched, as it's included by `gcc-13_gcc-13`.